pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -86,3 +90,25 @@ func IsReadyToReconcile(c client.Client, clustercontext *clusterd.Context, names
 	logger.Infof("%s: CephCluster %q found but skipping reconcile since Ceph health is %q", controllerName, cephCluster.Name, status.Health.Status)
 	return cephCluster.Spec, false, cephClusterExists, WaitForRequeueIfCephClusterNotReady
 }
+
+// REMOVE ME?
```

Comment:
still needed?

Reply:
Still needed by the `github.com/rook/rook/pkg/operator/ceph/disruption/machinedisruption` package.